code/obj/critter/sword.dm

```diff
@@ -0,0 +1,866 @@
+/* ================================================== */
+/* --- Syndicate Weapon: Orion Retribution Device --- */
+/* ================================================== */
+
+/obj/critter/sword
+	name = "Deep Space Beacon"
+	var/transformation_name = "Syndicate Locator Beacon"
+	var/true_name = "Syndicate Weapon: Orion Retribution Device"
+	desc = "A huge beacon, seemingly constructed for broadcasting long-range signals."
+	var/transformation_desc = "A huge beacon, seemingly constructed for baiting Nanotrasen personnel into thinking it's just a beacon."
+	var/true_desc = "An automated miniature doomsday device constructed by the Syndicate."
+	icon = 'icons/misc/retribution/SWORD/base.dmi'
+	icon_state = "beacon"
+	dead_state = "anchored"
+	death_text = "The Syndicate Weapon stops moving, leaving wreckage in it's wake."
+	pet_text = "tries to get the attention of"
+	angertext = "focuses on"
+	atk_text = "bumps into"
+	chase_text = "chases after"
+	crit_text = "slams into"
+	alpha = 0
+	atk_delay = 50
+	crit_chance = 25
+	health = 5400
+	bound_height = 96
+	bound_width = 96
+	layer = MOB_LAYER + 5
+	atkcarbon = 1
+	atksilicon = 1
+	flying = 1
+	generic = 0
+	seekrange = 128							//A perk of being a high-tech prototype - large detection range.
+	var/mode = 0							//0 - Beacon. 1 - Unanchored. 2 - Anchored.
+	var/cooldown = 0						//Used to prevent the SWORD from using abilities all the time.
+	var/used_ability = 0					//Used to only allow transforming after at least one ability has been used.
+	var/current_heat_level = 0				//Used to keep track of the SWORD's heat for Heat Reallocation.
+	var/transformation_triggered = false	//Used to check if the initial transformation has already been started or not.
+	var/rotation_locked = false				//Used to lock the SWORD's rotation in place. Or, at the very least, attempt to.
+	var/changing_modes = false				//Used to prevent some things during transformation sequences.
+	var/died_already = false				//Used to prevent spam-reporting the death of the SWORD.
+	var/past_destructive_rotation = null	//Used to prevent the SWORD from using Destructive Leap/Destructive Flight in the same direction twice in a row, at a 75% efficiency.
+	var/current_ability = null				//Used to keep track of what ability the SWORD is currently using.
+	var/stuck_location = null				//Used to prevent the SWORD from getting stuck too much.
+	var/stuck_timer = null					//Ditto.
+	var/image/glow							//The glow overlay.
```

Comment:
can you change these to use the self documenting code format

```dm
/// my comment
var/myvar
```

Reply:
Will do so in about an hour.

Reply:
Done.